apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -1311,6 +1312,8 @@ impl Telemetry {
                 }
             }
         } else {
+            println!("MISSING REPORTING");
```

Comment:
Does it still need to be there ? If you want to log something could you use our logging macros instead and use the right log level please

Reply:
Oof thanks for the catch @bnjjj, this was me just debugging, will remove.